04_numeronym/joel00wood/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func main() {
+	fmt.Println(numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronym(s string) string {
+	var out string
+	len := len(s)
+	midLen := len - 2
+	if midLen > 1 {
+		start := s[:1]
+		mid := strconv.Itoa(midLen)
+		end := s[len-1:]
+		out = start + mid + end
+	} else {
```

Comment:
purely philosophical / nitpick:

I tend to avoid fallback `else` wherever I can, ie. your `out` declaration could just be `out := s` at the top of the function and you could get rid of the `else`.

What are your thoughts?

Reply:
In terms of code readability that does make more sense. Whereby you don't need to wade through a more complex outcome to read through to the simpler one.

In the context of this function it seems particularly fitting given that the end result is still the input, whether it's been manipulated into a numeronym or not. I think part of my reasoning here was that in an earlier lab I got burned by not making the separation between input slice and output slice. But given that this isn't slices it seems reasonable to make this change.